.github/workflows/main-cmake.yml

```diff
@@ -128,9 +128,11 @@ jobs:
           run: |
             echo "Run ctest PATH: $PATH"
             export PATH=/c/Users/runneradmin/miniconda3/Library/lib:${PATH}
+            export GITHUB_PATH=/c/Users/runneradmin/miniconda3/Library/lib:${GITHUB_PATH}
```

Comment:
You need to do this instead: `echo "/c/Users/runneradmin/miniconda3/Library/lib" >> $GITHUB_PATH`, and possibly to replace the line above as well.

Reply:
Sorting back through previous pushes, that was failing to be reflected PATH/GITHUB_PATH when examined in various, later steps.  Hence the brute force approach I'm currently taking.  Determine that this is in fact the error, and then refine this.